### PR TITLE
Finish opensource lesson to resolve #2151

### LINKED ISSUE #2151: Finish Opensource Lesson
The Opensource lesson seems to be partially-written but not completely finished.

### Page where problem found?

[OpenSource.md
](https://github.com/Techtonica/curriculum/blob/16604ee37a62d852ffd8416cba3d6a435cce50d4/open-source/open-source.md)

### Type of problem

- Several links are placeholders
- List of companies in opensource is incomplete/probably not the most beginner-friendly right now
- Content could use a little more about social norms and ethos

### Suggested Solution

Somebody should finish this lesson! That can be me if you'd like, or somebody else. No strong preference, but I'd love to see us have a robust lesson here.

### PR DESCRIPTION
**Pull request by MaggieFero**
Finish the opensource lesson in our curriculum with special attention to problems described in issue #2151

### CODE REVIEW

File: open-source/open-source.md [hunk 1]
```diff
@@ -55,22 +70,10 @@ Contributing to Open Source projects offers many benefits:
...
-There are tons of projects out there! Once you find one you're interested in, go to the `Issues` section of the repo and [look for a `first-timers-only` label](http://www.firsttimersonly.com/). Not every project will have this label, but if you find an issue with that label, grab it and start working on it!
-
-_"I don't want to code outside of work!"_
-
-If you find an interesting project, your only contribution doesn't have to be just code. You could contribute to documentation, write blog posts, or even answer questions on Stack Overflow.
-
 ### Guided Practice
 
 [Operation Code](https://github.com/OperationCode/operationcode_frontend) could be an interesting place to start with enough set up instructions to get started. The goal of this exercise is to go through the steps of setting up an Open Source project on your computer and get it ready to run locally to be able to start fixing bugs.
```
Comment (MaggieFero) [inline open-source/open-source.md:79]: This repo is now-defunct, and the repo that replaced it is really inactive. @daaimah123 do you have a suggestion for a Javascript-based repo I could recommend instead? I do a lot more Python in opensource.

File: open-source/open-source.md [hunk 2]
```diff
@@ -55,22 +70,10 @@ Contributing to Open Source projects offers many benefits:
 
 - [Women Who Code](http://womenwhocode.github.io/#project_reviewers) (See 'Featured Repos' and 'How do we contribute to a project?' Sections)
```
Comment (MaggieFero) [inline open-source/open-source.md:71]: Women Who Code shut down 😢 I'm not sure which JS repos would be best to suggest instead. There's a CSS linter I mentioned in one instance where we previously spoke about Operation Code (now also defunct), but I don't really know them well enough to vouch. They seem extremely cool from an external review perspective, but I'm not currently in any JS project opensource communities. @daaimah123 Any suggestions for here?

**Review by MaggieFero — COMMENTED** [review]
I asked @daaimah123 for help coming up with replacement examples in a few places, and once we have those sections fixed this will be ready for review.

File: open-source/open-source.md (hunk 1 above)
Comment (daaimah123) [reply open-source/open-source.md:79]:
```suggestion
The goal of this exercise is to go through the steps of setting up an Open Source project on your computer and get it ready to run locally to be able to start fixing bugs.
```

It looks like there are many other maintained links on the page that I would want them pointed to that don't run the risk of projects becoming inactive. Let's remove this link

File: open-source/open-source.md (hunk 2 above)
Comment (daaimah123) [reply open-source/open-source.md:71]: Remove the reference, the others are okay.

**Review by daaimah123 — APPROVED** [review]
## LGTM 🥳

**Pull request merged**